ui/src/layout/Dashboard/CounterDisplay/index.js

```diff
@@ -1,19 +1,46 @@
-import React from 'react';
+import React, {useMemo} from 'react';
 import { useFetch } from 'hooks';
 import { formatNumber } from 'utils/utils';
 
+import COLORS from "../../../utils/scss_variables.module.scss";
+import { APIS } from "../../../utils/systemConsts";
+
 import './counter-display.scss';
 
-const CounterDisplay = ({url, title, background}) => {
+export const ScanCounterDisplay = () => {
+    const [{data, error, loading}] = useFetch(APIS.SCANS, {queryParams: {"$select": "state"}});
+
+    const completedScans = useMemo(() => {
+        let ret = 0;
+
+        data?.items.forEach((element) => {
+            if (element.state === "Aborted" || element.state === "Failed" || element.state === "Done") {
+                ret++
+            }
+        })
+
+        return ret
+    }, [data])
```

Comment:
This logic can be simplified:
```suggestion
    const completedScans = useMemo(
        () => data?.items.filter(item => item.state === "Aborted" || item.state === "Failed" || item.state === "Done").length ?? 0,
        [data]
    );
```